sdk/keyvault/keyvault-keys/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 4.0.0 (2019-10-30)
+
+- This release marks the general availability of the `@azure/keyvault-keys` package.
```

Comment:
@sadasant In https://github.com/Azure/azure-sdk-for-js/pull/5929, you accidentally updated the changelog for the previous version than the current version. Please create a new PR after this one gets merged to fix that.

Reply:
Same for secrets

Reply:
on my way!